app/views/layouts/admin_layout.html.erb

```diff
@@ -39,20 +39,25 @@
     </main>
   </div>
 
+  <% footer_links = [
+       {
+         href: "https://www.gov.uk/government/content-publishing",
+         text: "How to write, publish, and improve content"
+       }
+     ]
+  %>
+
+  <% footer_links.unshift({
+       href: Plek.new.external_url_for("support"),
+       text: "Raise a support request"
+     }) if user_signed_in?
+  %>
+
   <%= render "govuk_publishing_components/components/layout_footer", {
     navigation: [
       {
         title: "Support and feedback",
-        items: [
-          {
-            href: Plek.new.external_url_for("support"),
-            text: "Raise a support request"
-          },
-          {
-            href: "https://www.gov.uk/government/content-publishing",
-            text: "How to write, publish, and improve content"
-          }
-        ]
+        items: footer_links,
```

Comment:
```suggestion
        items: [
          *([
            href: Plek.new.external_url_for("support"),
            text: "Raise a support request"
          ] if user_signed_in?),
          {
            href: "https://www.gov.uk/government/content-publishing",
            text: "How to write, publish, and improve content"
          }
        ],
```

Does this work?

Reply:
That doesn't quite work as the item we need to add is a hash. I've given it an update to bring it closer to your idea - it's a bit code golf like but I think it's ok. Mind giving this a quick re-review?